.travis.yml

```diff
@@ -8,6 +8,7 @@ cache:
   - node_modules
 node_js:
   - '7'
+  - '8'
```

Comment:
Greenkeeper ops is not running on node 8 just yet, so we’d rather keep an engine update out of this PR until we move over the whole project.